client/src/app/site/motions/components/motion-change-recommendation/motion-change-recommendation.component.ts

```diff
@@ -104,14 +104,16 @@ export class MotionChangeRecommendationComponent {
     public createForm(): void {
         this.contentForm = this.formBuilder.group({
             text: [this.changeReco.text, Validators.required],
-            diffType: [this.changeReco.type, Validators.required]
+            diffType: [this.changeReco.type, Validators.required],
+            public: [!this.changeReco.internal]
```

Comment:
This is supposed to be the default value. I assume that should be true. suggests now that it's the opposite of the loaded changeReco.internal

Reply:
This form is also used when editing a change recommendation afterwards. In that case, the default value comes from the saved entry. The "!" comes from the fact that for the UI, I find a "set a checkbox to make it visible" more intuitive than a "set a checkbox to make in internal", while the backend uses "internal".